dm/loader/lightning.go

```diff
@@ -58,6 +58,7 @@ type LightningLoader struct {
 	toDB            *conn.BaseDB
 	toDBConns       []*DBConn
 	lightningConfig *lcfg.GlobalConfig
+	timeZone        string
```

Comment:
can we reuse `l.cfg.Timezone`?

Reply:
l.cfg.Timezone might be empty. timeZone is determined at `Init()`, but actually used in `Process` so we need a variable to store its value.